primitives/consensus/babe/src/lib.rs

```diff
@@ -364,20 +368,22 @@ sp_api::decl_runtime_apis! {
 		#[changed_in(2)]
 		fn configuration() -> BabeGenesisConfigurationV1;
 
-		/// Returns the slot that started the current epoch.
-		fn current_epoch_start() -> Slot;
+		/// Returns the slot that started the current session.
+		#[renamed("current_epoch_start", 3)]
+		fn current_session_start() -> Slot;
 
-		/// Returns information regarding the current epoch.
-		fn current_epoch() -> Epoch;
+		/// Returns information regarding the current session.
+		#[renamed("current_epoch", 3)]
+		fn current_session() -> Session;
 
-		/// Returns information regarding the next epoch (which was already
-		/// previously announced).
-		fn next_epoch() -> Epoch;
+		/// Returns information regarding the next session (which was already announced).
+		#[renamed("next_epoch", 3)]
+		fn next_session() -> Session;
```

Comment:
API version incremented to 3.

Added a `renamed` attribute with renaming version  for renamed methods